src/main/java/org/example/HomeWork1.java

```diff
@@ -0,0 +1,120 @@
+package main.java.org.example;
+
+import java.util.*;
+import java.util.stream.IntStream;
+
+public class HomeWork1 {
+
+    //Problem 1...
+    public int getMaxWordOccurrence(String text){
+
+        List<String> sentences;
+        List<Integer> sentenceLengths = new ArrayList<>();
+        //Here we are converting text to array of sentences, we are replacing . ? ! with _ symbol and then
+        //splitting this text into arrays with _ symbol.
+        sentences = Arrays.asList(text.replaceAll("[.?!]", "_").split("_"));
+
+        //here we are looping through each sentence and calculating and adding word count in each sentence
+        //with the help of splitting each word with " " and getting length
+        for (String sentence: sentences) {
+            sentenceLengths.add(sentence.trim().split(" ").length);
+        }
+
+        //here we are returning the maximum word occurrence in sentence
```

Comment:
Here also you don`t need this comment. It`s obvious what you are returning. Comments are good, but only when needed